Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '5')),
+    parameters([
+        stringParam(defaultValue: '26', description: 'Set 6LoWPAN Channel', name: 'CHANNEL'),
+        stringParam(defaultValue: '0xabcd', description: 'Set 6LoWPAN PAN ID', name: 'PAN_ID'),
+        stringParam(defaultValue: '1', description: 'Set Serial port to use for console', name: 'PADS'),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest" // TODO for now using the creator one
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                ],
+            ])
+        }
+        stage('Build') {
+            sh 'git submodule init "dev/ca8210" && git submodule update'
+            sh "cd examples/hello-world && make TARGET=mikro-e USE_CA8210=1 CHANNEL=${params.CHANNEL} PAN_ID=${params.PAN_ID} USE_SERIAL_PADS=${params.PADS}"
+            sh 'cd examples/hello-world && xc32-bin2hex hello-world.mikro-e && mv hello-world.hex ${WORKSPACE}/'
+            sh 'cd examples/ && git clone https://github.com/mtusnio/LetMeCreateIoT.git'
+            sh 'cd examples/LetMeCreateIoT/ && ./install.sh -p ../../'
+            sh "cd examples/LetMeCreateIoT/examples/accel/ && make && mv main.hex ${WORKSPACE}/accel.hex"
+            sh "cd examples/LetMeCreateIoT/examples/debug && make && mv main.hex ${WORKSPACE}/debug.hex"
+            sh "cd examples/LetMeCreateIoT/examples/joystick && make && mv main.hex ${WORKSPACE}/joystick.hex"
+            sh "cd examples/LetMeCreateIoT/examples/led_matrix && make && mv main.hex ${WORKSPACE}/led_matrix.hex"
+            sh "cd examples/LetMeCreateIoT/examples/matrix_server && make && mv main.hex ${WORKSPACE}/matrix_server.hex"
+            sh "cd examples/LetMeCreateIoT/examples/thermo3 && make && mv main.hex ${WORKSPACE}/thermo3.hex"
+            sh "cd examples/LetMeCreateIoT/examples/thermo3_client && make && mv main.hex ${WORKSPACE}/thermo3_client.hex"
+            sh "cd examples/LetMeCreateIoT/examples/udp_client && make && mv main.hex ${WORKSPACE}/udp_client.hex"
+            sh "cd examples/LetMeCreateIoT/examples/udp_client_mdns && make && mv main.hex ${WORKSPACE}/udp_client_mdns.hex"
+            sh "cd examples/LetMeCreateIoT/examples/udp_server && make && mv main.hex ${WORKSPACE}/udp_server.hex"
+            sh "cd examples/LetMeCreateIoT/tests/6lowpan && make && mv main.hex ${WORKSPACE}/6lowpan.hex"            
```

Comment:
Same as above perhaps adding mechanism for 'PREFIX=foo make install' or an install script would be better here so that devs can do the same thing?